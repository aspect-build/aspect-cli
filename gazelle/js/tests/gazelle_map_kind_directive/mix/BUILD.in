load("//:defs.bzl", "ts_my_project")

ts_my_project(
    name = "mix",
    srcs = ["m.ts"],
)