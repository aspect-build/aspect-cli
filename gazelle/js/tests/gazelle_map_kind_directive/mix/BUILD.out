load("//:defs.bzl", "ts_my_project")

ts_my_project(
    name = "mix",
    srcs = ["m.ts"],
)

ts_my_project(
    name = "mix_tests",
    testonly = True,
    srcs = ["m.spec.ts"],
)
